BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -256,13 +256,13 @@ Transform velocityTransform
         [KSPField(isPersistant = true, guiActive = false, guiActiveEditor = false, guiName = "#LOC_BDArmory_PIDAutoTuningRollRelevance", advancedTweakable = true,
             groupName = "pilotAI_PID", groupDisplayName = "#LOC_BDArmory_PilotAI_PID", groupStartCollapsed = true),
             UI_FloatRange(minValue = 0f, maxValue = 1f, stepIncrement = 0.01f, scene = UI_Scene.All)]
-        public float autoTuningRollRelevance = 0.5f;
+        public float autoTuningRollRelevance = 0.05f;
 
-        //AutoTuning Fast Response Relevance
-        [KSPField(isPersistant = true, guiActive = false, guiActiveEditor = false, guiName = "#LOC_BDArmory_PIDAutoTuningFastResponseRelevance", advancedTweakable = true,
-            groupName = "pilotAI_PID", groupDisplayName = "#LOC_BDArmory_PilotAI_PID", groupStartCollapsed = true),
-            UI_FloatRange(minValue = 0f, maxValue = 1f, stepIncrement = 0.01f, scene = UI_Scene.All)]
-        public float autoTuningFastResponseRelevance = 0.2f;
+        ////AutoTuning Fast Response Relevance
```

Comment:
I'd like to keep the "Fast Response Relevance" for now as it does affect some of my planes, e.g., the Curtiss P-40.